### PR TITLE
Use Use Graph v2 NuGet package in Key Vault PowerShell instead of ref…

### PR DESCRIPTION
**Pull request by pomortaz**
…erencing PowerShell AD code which is broken in Fairfax

```
 Graph calls (*AzureRmAD* cmdlets) in Fairfax are broken because of
 the access token with wrong audience that is used for authentication
 with Graph service. Key Vault has a dependency to these cmdlets and
 therefore broken if FF. This change is to remove the dependency to
 PowerShell implementation of Graph authentication and AzureRmAD
 internal code and directly reference the new version of the Graph SDK
 v2.
```

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@pomortaz__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pomortaz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
please upgrade GRaph RBAC SDK package to 1.8.0

**Comment by hovsepm** [comment]
please add on-demand job execution link to PR comments.

File: setup/azurecmdfiles.wxi [hunk 1]
```diff
@@ -799,6 +799,9 @@
                         <Component Id="cmp5835D87A5A94F44EAAAE61BCA33C1D80" Guid="*">
                             <File Id="filEC64A002E6CBA598F7148F816DB05F11" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.DataLakeAnalytics\Microsoft.Azure.Commands.DataLakeAnalytics.dll" />
                         </Component>
+                        <Component Id="cmp96C621A371FF4A410548AAB167869D94" Guid="*">
+                            <File Id="filC7912B6B895CF7DF00A7C7F821AE40BF" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.DataLakeAnalytics\Microsoft.Azure.Commands.DataLakeAnalytics.dll-help.xml" />
```
Comment (hovsepm) [inline setup/azurecmdfiles.wxi]: you should not have DataLake dependency changes. please run git clean -dfx and then run BuildInstaller.ps1 from tools folder again.

**Comment by stankovski** [comment]
http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/435/

**Comment by stankovski** [comment]
http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-msi/592/

**Pull request merged**